docs/tutorials/developer-journey/level-1/1.4-using-cycles.md

```diff
@@ -0,0 +1,229 @@
+# 1.4: Acquiring and using cycles
+
+## Overview
+
+If you recall from the previous module, [Internet Computer terminology](../level-0/02-ic-terms.md), cycles are used to measure the resources, such as memory, storage, and compute power, that are used by a canister. When a canister is deployed on the mainnet, cycles are 'charged' for every action that a canister performs.
+
+To obtain cycles, the IC's utility token, ICP, can be converted into cycles and transferred into a canister to be used to pay for that canister's consumed resources. ICP can be converted to cycles using a conversion rate where the current price of ICP measured is in [XDR](https://internetcomputer.org/docs/current/references/glossary#xdr), and one trillion cycles corresponds to one XDR. 
+
+Since cycles are not a currency and are only used to pay for a canister's consumed resources, developers manage the distribution of cycles through a special canister called a **cycles wallet**. When a canister needs to use the cycles stored in the cycles wallet, the canister's operations are executed using the canister principal of the cycles wallet, rather than your user principal. 
+
+For local canister execution, the SDK automatically creates a default cycles wallet in every project, and operations performed using cycles are done in the background. In a production environment with canisters deployed on the mainnet, canisters will need to have cycles explicitly registered and transferred into them. Production canisters will also require principals to be configured to act as **custodians**, which are principals that have permission to send and receive cycles for the canister. 
+
+In this tutorial, we'll cover how to set up a cycles wallet and fill it with cycles to use. Then, in the next tutorial we'll deploy a canister to the mainnet using those cycles. 
+
+## Prerequisites
+
+Before you start, verify that you have set up your developer environment according to the instructions in [0.3: Developer environment setup](../level-0/03-dev-env.md).
+
+## Creating a developer identity
+
+When you initially use `dfx`, a default developer identity is created. This identity is a `principal` data type, often referred to as your **principal identifier**. This identity is similar to a Bitcoin or Ethereum wallet address that you would use to interact with those ecosystems. 
+
+However, your developer identity principal is not the same as your account identifier that is specified in the ledger. Your identity principal and your account identifier are related, but they use different formats.
+
+In this tutorial, we'll create a new identity principal with `dfx`, which we'll use to obtain cycles and deploy a cycles wallet. 
+
+First, assure that `dfx` is running; if not, use the following command to start it:
+
+```
+dfx start --background
+```
+
+Next we will create a new user identity principal with the command:
+
+```
+dfx identity new DevJourney
```

Comment:
I'd also add a caution here that people should create a backup of their identity. It often happens that people use their identity that was associated with their cycles wallet that they got with a coupon.